### DISCUSSION TITLE
Cannot exit dosemu after autoexec.bat error

### DISCUSSION THREAD

**Discussion by nelgin** [dosemu2/dosemu2#1711] [General]
**Describe the bug**
I have created an fdppconf.sys and config.sys in my drive_c directory. When I run dosemu.bin -f /sbbs/ctrl/dosemu.conf I get the following error:

Cannot open C:\AUTOEXEC.BAT
Batch file aborted - C:\AUTOEXEC.BAT, line 1

At this stage, neither exit or exitemu will exit the program. This is a huge security issue on my BBS because the batch file that runs dosemu expects to be able to use exitemu otherwise it could leave a user at a dos prompt with access to the bbs files.

exit just returns to the prompt
exitemu says no such command

**To Reproduce**
See above

**Attach the log**
[debug.log.gz](https://github.com/dosemu2/dosemu2/files/9018314/debug.log.gz)

**Additional info**
Same configuration as before with a few tweaks to test stuff.
[dosemu.conf.txt](https://github.com/dosemu2/dosemu2/files/9018338/dosemu.conf.txt)

**Comment by andrewbird** [comment]
Hello,
  fdpp has its own versions of `autoexec.bat` and `config.sys` called `fdppauto.bat` and `fdppconf.sys` respectively. It expects to use these as shipped to setup things like the location of the dosemu tools drive/directory and other important config. Obviously users may want their own specific settings to be applied on startup, for that create files called `c:\userhook.bat` and `c:\userhook.sys` to include your changes. As you've found it is possible to alter this standard setup, but expect things to break in unexpected ways.

BTW you have a typo in your dosemu.conf `$_lp2 = ""` it's missing a `t`

**Comment by stsp** [comment]
Moved to discussions.

**Comment by nelgin** [comment]
Good information, thank you. There's a bit of conflicting information out there. This clears it up.
